.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+package-lock.json
```

Comment:
i thought these would be better in the project, but then it would be interesting to see how people deal with this